Configuration/TypoScript/setup.txt

```diff
@@ -1,13 +1,15 @@
-page.includeCSS.aimeos-common = EXT:aimeos/Resources/Public/Themes/elegance/common.css
-page.includeCSS.aimeos = EXT:aimeos/Resources/Public/Themes/elegance/aimeos.css
+page.includeCSS.aimeos-common = {$plugin.tx_aimeos.theme.css.aimeos-common}
```

Comment:
And the changes at setup.txt to get the values set before at constants.txt. It's also possible to override such values at other extensions like the dist at constants.txt level!